_examples/ssh-remoteforward/remoteforward.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"io"
+	"log"
+
+	"github.com/gliderlabs/ssh"
+)
+
+func main() {
+	ssh.Handle(func(s ssh.Session) {
+		io.WriteString(s, "Remote forwarding available...\n")
```

Comment:
Does this mean that forwarding is on by default? (Is that a good idea?)

Reply:
Right, prob not. Now see https://github.com/gliderlabs/ssh/issues/68